lib/hubspot/event.rb

```diff
@@ -0,0 +1,21 @@
+require 'hubspot/utils'
+
+module Hubspot
+  #
+  # HubSpot Events HTTP API
+  #
+  # {https://developers.hubspot.com/docs/methods/enterprise_events/http_api}
+  #
+  class Event
+    POST_EVENT_PATH = '/v1/event'
+
+    class << self
+      def complete(event_id, email, options = {})
```

Comment:
I think `trigger` would be a better word here (and in `EventConnection`) that more closely aligns with the documentation and intent of the API. The documentation refers to this endpoint as:
>Trigger a custom HubSpot event using an HTTP GET call.

Reply:
will do!